src/vs/workbench/contrib/searchEditor/browser/searchEditorActions.ts

```diff
@@ -178,11 +179,17 @@ export const createEditorFromSearchResult =
 		const instantiationService = accessor.get(IInstantiationService);
 		const labelService = accessor.get(ILabelService);
 		const configurationService = accessor.get(IConfigurationService);
-		const sortOrder = configurationService.getValue<ISearchConfigurationProperties>('search').sortOrder;
-
+		const fileService = accessor.get(IFileService);
 
 		telemetryService.publicLog2('searchEditor/createEditorFromSearchResult');
 
+		const sortOrder = configurationService.getValue<ISearchConfigurationProperties>('search').sortOrder;
+		if (sortOrder === SearchSortOrder.Modified) {
+			// Ensure all matches have retrieved their file stat
+			const files = searchResult.matches().filter(f => !f.fileStat).map(f => f.resolveFileStat(fileService));
```

Comment:
Did you find a case where this was needed? I'd expect the stats to all be resolved based from the existing search.

Reply:
yeah you are right, I'll revert this changes